src/data/fundingSpace.ts

```
@@ -0,0 +1,28 @@
+import { FundingSpace, Organization } from '../entity';
+import { AgeGroup, FundingSource } from '../../client/src/shared/models';
+import { FUNDING_SOURCE_TIMES } from '../../client/src/shared/constants';
+
+export const getFakeFundingSpaces = (
+  organization: Organization
+): FundingSpace[] => {
+  let fundingSpaces = [];
+  Object.values(AgeGroup).map((ageGroup) => {
+    for (const source of Object.values(FundingSource)) {
+      const match = FUNDING_SOURCE_TIMES.find((fst) =>
+        fst.fundingSources.includes(source)
+      );
+      if (match) {
+        fundingSpaces = fundingSpaces.concat(
+          match.fundingTimes.map((fundingTime) => ({
+            ageGroup,
+            capacity: 10,
```

Should we randomize this as well?

---

You're right that we probably should make all of this data a little closer to our actual data-- I basically got lazy and wanted to finish the PR.  I think maybe we can include "more realistic fake data" as a task for testing tix, since we also need to fake all of the different combinations of missing data, error states, etc